### ISSUE TITLE
building gmp fails on ARM

### ISSUE THREAD

**Issue by notwa** [mxe/mxe#951]
```
checking build system type... Invalid configuration `neon': machine `neon' not recognized
```

this is actually an old bug in the `config.guess` of the gmp package.
it can be resolved by invoking configure with ```--build="`config.guess`"``` as done with other packages.

**Comment by tonytheodore** [comment]
Should be fixed as soon as #925 is merged, the [new rule](https://github.com/tonytheodore/mxe/blob/08e0dc6de91dc2f42d38215e92e8c475a1a1559f/src/gmp.mk) uses the standard `$(MXE_CONFIGURE_OPTS)` which includes `--build`.

**Comment by notwa** [comment]
cool, thanks. merging #925 fixed it.
